src/overlays/actors/ovl_En_Firefly/z_en_firefly.c

```diff
@@ -9,7 +16,33 @@ void EnFirefly_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnFirefly_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnFirefly_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void EnFirefly_FlyIdle(EnFirefly* this, GlobalContext* globalCtx);
+void EnFirefly_Fall(EnFirefly* this, GlobalContext* globalCtx);
+void EnFirefly_SetupDie(EnFirefly* this);
+void EnFirefly_Die(EnFirefly* this, GlobalContext* globalCtx);
+void EnFirefly_SetupDiveAttack(EnFirefly* this);
+void EnFirefly_DiveAttack(EnFirefly* this, GlobalContext* globalCtx);
+void EnFirefly_Rebound(EnFirefly* this, GlobalContext* globalCtx);
+void EnFirefly_SetupFlyAway(EnFirefly* this);
+void EnFirefly_FlyAway(EnFirefly* this, GlobalContext* globalCtx);
+void EnFirefly_Stunned(EnFirefly* this, GlobalContext* globalCtx);
+void EnFirefly_SetupPerch(EnFirefly* this);
+void EnFirefly_Perch(EnFirefly* this, GlobalContext* globalCtx);
+void EnFirefly_SetupDisturbDiveAttack(EnFirefly* this);
+void EnFirefly_DisturbDiveAttack(EnFirefly* this, GlobalContext* globalCtx);
+
+typedef enum {
+    /* 0 */ KEESE_FIRE,
+    /* 3 */ KEESE_NORMAL = 3,
+    /* 4 */ KEESE_ICE
+} KeeseCurrentType;
+
+typedef enum {
+    /* 0 */ KEESE_AURA_NONE,
+    /* 1 */ KEESE_AURA_FIRE,
+    /* 2 */ KEESE_AURA_ICE
+} KeeseAuraType;
```

Comment:
Would it be better for the enums to go in the header file?

Reply:
These can only be used internally, so seem better in the C file to me.